multinode-demo/validator.sh

```diff
@@ -285,7 +285,7 @@ setup_validator_accounts() {
   return 0
 }
 
-rpc_url=$($solana_gossip rpc-url --entrypoint "$gossip_entrypoint")
+rpc_url=$($solana_gossip rpc-url --timeout 180 --entrypoint "$gossip_entrypoint")
```

Comment:
Yeah, was hard to debug.... Well, this depended on exact build timing. ;)

Greatly increase from the default `15` seconds to `180` to prevent another sorrow. ;)